commands/system/help.js

```diff
@@ -1,83 +1,162 @@
+      let num = 0;
+      let output = "";
+      const pg = Number(pagenumber);
+      for (const c of sorted.values()) {
+        if (c.help.category.toLowerCase() === type.toLowerCase()) {
+          if (c.help.category === "NSFW" && !message.channel.nsfw) continue;
+          if (num < perpage * pg && num > perpage * pg - (perpage + 1)) {
+            if (level < this.client.levelCache[c.conf.permLevel]) continue;
+            output += `\n\`${message.settings.prefix + c.help.name}\` | ${c.help.description.length > 80 ? `${c.help.description.slice(0, 80)}...` : c.help.description}`;
+          }
+          num++;
+        }
+      }
+      reactions.users.remove(message.author);
+      if (direction === "forward") {
+        if (pg > Math.ceil(num / perpage)) return;
+      }
+      if (direction === "backward") {
+        if (pg === 0) return;
+      }
+      const helpembed = new MessageEmbed()
+        .setTitle(`Page ${pg}/${Math.ceil(num / perpage)} for ${type.toProperCase()}`)
+        .setDescription(`A list of commands in the ${type} category.\n(Total of ${num} commands in this category)\n\nTo get help on a specific command do \`${message.settings.prefix}help <command>\``)
+        .addField("Commands", output)
+        .setColor(message.guild.me.roles.highest.color || 5198940)
+        .setFooter(`Requested by ${message.author.tag}`, message.author.displayAvatarURL());
+      await helpmessage.edit(helpembed);
+    };
+  }
+
+  async run(message, [type, page = 1], level) {
+    let num = 0;
+    const helpembed = new MessageEmbed()
       .setTimestamp()
       .setColor(message.guild.me.roles.highest.color || 5198940)
-      .setFooter(`Requested by ${message.author.tag}`, message.author.avatarURL()); 
+      .setFooter(`Requested by ${message.author.tag}`, message.author.displayAvatarURL());
 
-    let currentCategory = "";
-    const sorted = this.client.commands.sort((p, c) => p.help.category > c.help.category ? 1 :  p.help.name > c.help.name && p.help.category === c.help.category ? 1 : -1 );
+    const currentCategory = "";
+    const last = [];
+    const sorted = this.client.commands.sort((p, c) => (p.help.category > c.help.category ? 1 : p.help.name > c.help.name && p.help.category === c.help.category ? 1 : -1));
     if (!type) {
       const description = `Command category list\n\nUse \`${message.settings.prefix}help <category>\` to find commands for a specific category`;
-      const output = sorted.filter(c => !(level < 10 && c.help.category == "Owner") || !(c.help.category === "NSFW" && !message.channel.nsfw)).map(c => {
-        const cat = c.help.category.toProperCase();
+      const output = sorted.map((c) => {
+        const cat = c.help.category;
         if (currentCategory !== cat && !type) {
-          currentCategory = cat;
-          return `\n\`${message.settings.prefix}help ${cat.toLowerCase()}\` | Shows ${cat} commands`;
+          if (last.includes(cat.toLowerCase())) return;
```

Comment:
Instead of `toLowerCase`ing it every time you use it, just do `const cat = c.help.category.toLowerCase();`
and add `.toLowerCase()` to the check below (to currentCategory)